### PR TITLE
Trigger resize on interactive/complete/DOMContentLoaded

### PR DESCRIPTION
**Pull request by forberg**
Much faster resize in all modern browsers.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -85,10 +85,25 @@ var bootStrap = function (options) {
...
+    function resizeHandler(name) {
         var size = childrenSize(gardr.container);
-        pluginApi.trigger('banner:rendered', size);
+        pluginApi.trigger('banner:' + name, size);
         com.rendered(size);
+    }
+
+    eventListener.add(doc, 'readystatechange', function readyStateChangeHandler() {
+        if (/interactive|complete/.test(doc.readyState)) {
+            resizeHandler(doc.readyState);
```
Comment (sveisvei) [inline lib/index.js]: Isnt `complete` the same as load? ( https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState)

Comment (gregersrygg) [inline lib/index.js]: `complete` is the same as load, and `interactive is the same as `DOMContentLoaded` (even though IE might behave differently). So do you actually need readystatechange at all?

File: lib/index.js [hunk 2]
```diff
@@ -85,10 +85,25 @@ var bootStrap = function (options) {
...
+        }
+    });
+
+    // Faster in some IE versions
+    eventListener.add(doc, 'DOMContentLoaded', function domContentLoadedHandler() {
+        resizeHandler('domcontentloaded');
```
Comment (gregersrygg) [inline lib/index.js]: I think `interactive` or `ready` would be a better event name.

**Comment by gregersrygg** [comment]
Looks like a nice addition, but it needs to have good test coverage before merging.

**Pull request closed without merging**